### PR TITLE
Add gitignore and static icons

### PR DESCRIPTION
**Pull request by 526avijitgupta**

### CODE REVIEW

File: static/icons/amazonaws.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2.08.271.12.566.215.878.279.319.064.63.096.95.096.502 0 .894-.088 1.165-.264a.86.86 0 0 0 .415-.758.777.777 0 0 0-.215-.559c-.144-.151-.416-.287-.807-.415l-1.157-.36c-.583-.183-1.014-.454-1.277-.813a1.902 1.902 0 0 1-.4-1.158c0-.335.073-.63.216-.886.144-.255.335-.479.575-.654.24-.184.51-.32.83-.415.32-.096.655-.136 1.006-.136.175 0 .359.008.535.032.183.024.35.056.518.088.16.04.312.08.455.127.144.048.256.096.336.144a.69.69 0 0 1 .24.2.43.43 0 0 1 .071.263v.375c0 .168-.064.256-.184.256a.83.83 0 0 1-.303-.096 3.652 3.652 0 0 0-1.532-.311c-.455 0-.815.071-1.062.223-.248.152-.375.383-.375.71 0 .224.08.416.24.567.159.152.454.304.877.44l1.134.358c.574.184.99.44 1.237.767.247.327.367.702.367 1.117 0 .343-.072.655-.207.926-.144.272-.336.511-.583.703-.248.2-.543.343-.886.447-.36.111-.734.167-1.142.167zM21.698 16.207c-2.626 1.94-6.442 2.969-9.722 2.969-4.598 0-8.74-1.7-11.87-4.526-.247-.223-.024-.527.272-.351 3.384 1.963 7.559 3.153 11.877 3.153 2.914 0 6.114-.607 9.06-1.852.439-.2.814.287.383.607zM22.792 14.961c-.336-.43-2.22-.207-3.074-.103-.255.032-.295-.192-.063-.36 1.5-1.053 3.967-.75 4.254-.399.287.36-.08 2.826-1.485 4.007-.215.184-.423.088-.327-.151.32-.79 1.03-2.57.695-2.994z"/></svg>
```
Comment (yashmehrotra) [inline static/icons/amazonaws.svg:1]: Remove ? no one's gonna use "AWS" :P

Comment (526avijitgupta) [reply static/icons/amazonaws.svg]: Removed

File: exampleSite/content/posts/theme-documentation-advanced.md [hunk 2]
```diff
@@ -76,7 +103,7 @@ The goal of this feature is to give the user more control over the theme. It's f
 
 Add your own JavaScript or CSS by putting them in the `static/` folder and importing them into your HTML.
 
-```toml
+```markdown
```
Comment (yashmehrotra) [inline exampleSite/content/posts/theme-documentation-advanced.md:106]: no, its toml !!!

File: static/js/main.js [hunk 3]
```diff
@@ -12,6 +12,11 @@ function ready() {
     feather.replace({ 'stroke-width': 1, width: 20, height: 20 });
     setThemeByUserPref();
 
+    // Elements to inject
+    const svgsToInject = document.querySelectorAll('img.svg-inject');
+    // Do the injection
```
Comment (yashmehrotra) [inline static/js/main.js:17]: Thank you captain obvious 🖖🏽

File: exampleSite/content/posts/theme-documentation-advanced.md (hunk 2 above)
Comment (526avijitgupta) [reply exampleSite/content/posts/theme-documentation-advanced.md]: Fixed

File: static/js/main.js (hunk 3 above)
Comment (526avijitgupta) [reply static/js/main.js:17]: Copy pasted code FTW :D

**Pull request merged**